pkg/vault/create.go

```diff
@@ -80,8 +85,24 @@ func (v *vaultImpl) CreateVault(d string, deviceShards [][]byte, dscPub string,
 		return DefaultVaultService, fmt.Errorf("error creating vault: %s", errRes.Message)
 	}
 
+	env_path := filepath.Join(projectpath.Root, ".env")
+
+	var uri string
+	// by default use .env if it exists
+	_, err = os.Stat(env_path)
+	if errors.Is(err, os.ErrNotExist) {
+		uri = "https://vault.sonr.ws"
+	} else {
+		err = godotenv.Load(env_path)
+		if err != nil {
```

Comment:
If there's an error at this stage, `uri` is unlikely to get a value, so it should default still to `https://vault.sonr.ws`

Reply:
I added a function that gets the uri, and instead of exiting with error I simply log the error and return a default value for uri.